### PR TITLE
Session push

### PR DESCRIPTION
**Pull request by chistel**
push a new value into a session array.

### CODE REVIEW

File: system/Session/Session.php [hunk 1]
```diff
@@ -511,7 +511,22 @@ public function get(string $key = null)
...
+	/**
+	 * Push a value into a session array.
+	 *
+	 * @param  string  $key
+	 * @param  mixed   $value
+	 * @return void
+	 */
+	public function push($key, $value)
+	{
+		if ($this->has($key))
```
Comment (lonnieezell) [inline system/Session/Session.php]: Please change to an early out design: 

```
if (! $this->has($key)) return;

$array = $this->get($key);
$array = array_merge($array, $value);
$this->set($key, $array);
```

File: system/Session/Session.php [hunk 2]
```diff
@@ -511,7 +511,22 @@ public function get(string $key = null)
...
+	 * @param  string  $key
+	 * @param  mixed   $value
+	 * @return void
+	 */
+	public function push($key, $value)
+	{
+		if ($this->has($key)) 
+		{
+			$array = $this->get($key);
+			$array = array_merge($array, $value);
```
Comment (lonnieezell) [inline system/Session/Session.php]: Should have a check to ensure it's an array or throw an exception. I realize that array_merge will throw it's own language, but this gives us a chance to customize the message.

Comment (chistel) [reply system/Session/Session.php]: ok changes would be made

File: system/Session/Session.php (hunk 1 above)
Comment (chistel) [reply system/Session/Session.php]: ok

Comment (chistel) [reply system/Session/Session.php]: not getting my way with the test, not so go at it. mind putting me true?? i use windows.

Comment (jaynarayan89) [reply system/Session/Session.php]: HAve you read https://github.com/bcit-ci/CodeIgniter4/tree/develop/tests

Comment (chistel) [reply system/Session/Session.php]: @jaynarayan89  tried that, i keep getting no test executed.

**Comment by jim-parry** [comment]
Being picky, but we expect commits to be GPG-signed :)

**Comment by lonnieezell** [comment]
This hasn't been touched in 4+ months and I'm not convinced it adds much over the existing `set()` command. Closing.

**Pull request closed without merging**